collectors/apps.plugin/apps_groups.conf

```diff
@@ -338,3 +338,10 @@ filecoin: lotus* lotus-miner* lotus-worker*
 solana: solana*
 web3: *hardhat* *ganache* *truffle* *brownie* *waffle*
 terra: terra* mantle*
+
+# -----------------------------------------------------------------------------
+# chaos engineering tools
+
+stress-ng: stress-ng*
+gremlin: gremlin*
```

Comment:
It doesn't work w/o `*`?

Reply:
stress-ng will span other stress-ng-foo processes. 

gremlin* covers gremlind and then gremlin where gremlin is processes spawned by gremlin during an attack. 

didnt think any real need to split out gremlin and gremlind

Reply:
@andrewm4894 what about [stress](https://linux.die.net/man/1/stress)? Perhaps group both `stress` and `stress-ng` into one group?

Reply:
Asking about it, because I used it earlier today when was testing https://github.com/netdata/netdata/pull/12164

Reply:
hmm could make sense for stress and stress-ng maybe to be same group called "stress" maybe

Reply:
done